src/v/archival/ntp_archiver_service.cc

```diff
@@ -87,10 +87,10 @@ ntp_archiver::ntp_archiver(
 
 void ntp_archiver::run_sync_manifest_loop() {
     vassert(
-      !_sync_manifest_loop_started,
+      _sync_manifest_loop_state != loop_state::started,
```

Comment:
nit: I think we can also add here an assert that `_upload_loop` is not started since ntp_archiver can have only one of the loops running

Reply:
good point, thanks. added condition to both start functions, please review